data/strings/strings.txt

```diff
@@ -27019,28 +27019,33 @@
     comment = Text on car placeholder screen that maps are shown on the phone screen
     tags = android
     en = You are now using Organic Maps on the phone screen
+    fr = Vous utilisez maintenant Organic Maps sur l’écran du téléphone
     ru = Сейчас Вы используете Organic Maps на экране телефона
 
   [aa_used_on_car_screen]
     comment = Text on phone placeholder screen that maps are shown on the car screen
     tags = android
     en = You are now using Organic Maps on the car screen
+    fr = Vous utilisez maintenant Organic Maps sur l'écran de la voiture
     ru = Сейчас Вы используете Organic Maps на экране автомобиля
 
   [aa_connected]
     comment = Android Auto connected
     tags = android
     en = You are connected to Android Auto
+    fr = Vous êtes connecté à Android Auto
     ru = Вы подключены к Android Auto
 
   [aa_continue_on_phone]
     comment = Button to show maps on the phone screen instead of a car
     tags = android
     en = Continue on phone
+    fr = Continuer sur le téléphone
     ru = Продолжить в телефоне
 
   [aa_continue_in_car]
     comment = Button to show maps on the car screen instead of a phone. Must be no more than 18 symbols!
```

Comment:
Is it AA limit?

Reply:
Yes :(

Reply:
I suggest `fr = Aller sur le téléphone`

Reply:
@AndrewShkrob can this check be added to our strings linter?

Reply:
It appears that I might have confused some details. This button is meant to be utilized on the device's screen, hence there should be no limit. The restriction regarding the button limit should pertain to the button within AA instead. I will take another look to ensure clarity.

Reply:
Everything here is correct. Just confusing comment. I updated it in https://github.com/organicmaps/organicmaps/pull/5843